sopel-modules/rat-facts.py

```diff
@@ -253,13 +254,7 @@ def cmd_fact_import(bot, trigger):
 
     @require_overseer('Sorry, but you need to be an overseer or higher to execute this command.')
```

Comment:
Since it's an open page on /frkb anyway, we could probably bring this down to rat perm level.